lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -85,7 +85,10 @@ class Core
     "-k" => [ true,  "Keep (include) arg lines at start of output."   ],
     "-c" => [ false, "Only print a count of matching lines."          ])
 
-
+  @@history_opts = Rex::Parser::Arguments.new(
+    "-h" => [ false, "Help banner."                                   ],
+    "-a" => [ false, "Show all commands in history"                ],
```

Comment:
for consistency with the other options, please add a period to the end of this command